### ISSUE TITLE
Zuul: configure ribbon server list for specific route

### ISSUE THREAD

**Issue by vasilievip** [spring-cloud/spring-cloud-netflix#1132]
1. Can't find any metrics produced by SimpleHostRoutingFilter, it also doesn't wrap http call with hystrix
2. I'm trying to configure ribbon for specific route to use static IP list in order to get metrics and hystrix enabled
3. It seems that configuration per route is ignored

```
zuul.routes.sample=/sample/**
sample.ribbon.listOfServers=localhost:9010
sample.ribbon.NIWSServerListClassName=com.netflix.loadbalancer.ConfigurationBasedServerList
sample.ribbon.NFLoadBalancerRuleClassName=com.netflix.loadbalancer.RoundRobinRule
sample.ribbon.eureka.enabled=false
```

```
2016-06-24 18:52:44 INFO  BaseLoadBalancer:185 - Client:sample instantiated a LoadBalancer:DynamicServerListLoadBalancer:{NFLoadBalancer:name=sample,current list of Servers=[],Load balancer stats=Zone stats: {},Server stats: []}ServerList:null
```
Am I missing something?

P.S.: other routes do use eureka, I assume route specific ribbon client configuration is ignored by some reason...

**Comment by spencergibb** [comment]
`SimpleHostRoutingFilter` doesn't use eureka or hystrix.

See docs https://github.com/spring-cloud/spring-cloud-netflix/blob/master/docs/src/main/asciidoc/spring-cloud-netflix.adoc#embedded-zuul-reverse-proxy

```
zuul:
  routes:
    sample:
      path: /sample/**
      url: http://localhost:9010
```

Support for setting a server list via properties was just merged yesterday in 7dc8b86001ea6cb32b7e70b46ea716b0b6dfd0a8

**Comment by vasilievip** [comment]
updated cloud netflix to snapshot and it works. Thanks!
